README.md

```diff
@@ -107,24 +105,51 @@ or used interactively from rviz using the MotionPlanning GUI plugin.
   ```
 
 
-* For a first test, run the Moveit-created demo launch. Once rviz is running,
-  enable the motion planning plugin, then select one of the end effectors
-  of you robot. Rviz should show an 6-D (position and orientation)
-  interactive marker for the selected end-effector(s).
-  Move the interactive marker and watch bio_ik calculating poses for your robot.
+* For a first test, follow the [Quickstart in Rviz](https://moveit.picknik.ai/galactic/doc/tutorials/quickstart_in_rviz/quickstart_in_rviz_tutorial.html)
+  MoveIt2 tutorial. Before building your workspace, be sure to replace the kinematics solver for the Panda robot in
+  `<colcon workspace>/src/moveit_resources/panda_moveit_config/config/kinematics.yaml` as shown below:
+
 
-  If you also installed the bio_ik demo (see below), you should be able
-  to run one of the predefined demos:
+  ```yaml
+  panda_arm:
+    kinematics_solver: bio_ik/BioIKKinematicsPlugin
+    kinematics_solver_search_resolution: 0.005
+    kinematics_solver_timeout: 0.005
+    kinematics_solver_attempts: 1
   ```
-    roslaunch pr2_bioik_moveit demo.launch
-    roslaunch pr2_bioik_moveit valve.launch
-    roslaunch pr2_bioik_moveit dance.launch
+
+  To use a solver class besides the default `bio2_memetic`, the ROS param `mode` must be set to one
+  of the options [below](#disabling-global-optimization) (e.g. 'gd_c'). To do so for the Quickstart
+  in Rviz tutorial, edit the rviz node within `<colcon workspace>/src/moveit2_tutorials/doc/tutorials/quickstart_in_rviz/launch/demo.launch.py` as such:
+
+  ```python
+  rviz_node_tutorial = Node(
+    package="rviz2",
+    executable="rviz2",
+    name="rviz2",
+    output="log",
+    arguments=["-d", rviz_empty_config],
+    parameters=[
+        robot_description,
+        robot_description_semantic,
+        ompl_planning_pipeline_config,
+        kinematics_yaml,
+        {"mode": "gd_c"} # use gd_c solver instead of default "bio2_memetic"
```

Comment:
shouldn't this be in the group namespace?

Reply:
It can be -- currently the kinematics plugin doesn't use the group namespace for any of the parameters, but I suppose it should. I'll go ahead an make that change

Reply:
I've changed all the parameters to be prefixed with `robot_description_kinematics.[group_name]`. This is consistent with how parameters are set when retrieved from `kinematics.yaml`